frontend/public/manifest.json

```diff
@@ -9,12 +9,12 @@
   "description": "Jabber Application",
   "icons": [
     {
-      "src": "/img/icons/icon-512.png",
+      "src": "/img/icons/icon-512-pink.png",
```

Comment:
Wrong image name

Reply:
Will fix it soon

Reply:
remove old, and use only these

Reply:
you also need to change apple icon, social icon, favicon, svg icon